x/auctions/keeper/bid.go

```diff
@@ -0,0 +1,44 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/evmos/evmos/v19/utils"
+	"github.com/evmos/evmos/v19/x/auctions/types"
+)
+
+// GetHighestBid gets the highest bid
+func (k *Keeper) GetHighestBid(ctx sdk.Context) *types.Bid {
```

Comment:
Is there a specific reason why we're returning here a pointer instead of the struct?

Reply:
Nothing in particular we can use the struct directly here.